scripts/cluster/setup_master_node.sh

```diff
@@ -60,4 +60,15 @@ fi
 kubectl apply --filename $ROOT/configs/knative_yamls/serving-default-domain.yaml
 
 kubectl apply --filename https://github.com/knative/net-istio/releases/download/v0.19.0/release.yaml
-kubectl --namespace istio-system get service istio-ingressgateway
+
+# install knative eventing
+kubectl apply --filename https://github.com/knative/eventing/releases/download/v0.21.0/eventing-crds.yaml
```

Comment:
please make v0.21.0 a variable that is used for the stock serving and eventing.

Reply:
done